crates/node-core/src/args/log_args.rs

```diff
@@ -98,21 +98,31 @@ impl LogArgs {
 
     /// Initializes tracing with the configured options from cli args.
     pub fn init_tracing(&self) -> eyre::Result<Option<FileWorkerGuard>> {
+        // Create a new tracer instance.
         let mut tracer = RethTracer::new();
 
+        // Configure stdout logging based on cli args.
         let stdout = self.layer(self.log_stdout_format, self.log_stdout_filter.clone(), true);
         tracer = tracer.with_stdout(stdout);
 
+        // Add Journald logging if enabled.
         if self.journald {
             tracer = tracer.with_journald(self.journald_filter.clone());
         }
 
+        // Add file logging if max files limit is greater than 0.
         if self.log_file_max_files > 0 {
             let info = self.file_info();
             let file = self.layer(self.log_file_format, self.log_file_filter.clone(), false);
             tracer = tracer.with_file(file, info);
         }
 
+        // Set the global default subscriber with env filter.
+        tracing::subscriber::set_global_default(
+            FmtSubscriber::builder().with_env_filter(EnvFilter::from_default_env()).finish(),
+        )?;
+
+        // Initialize the tracer and return the guard.
         let guard = tracer.init()?;
```

Comment:
hmm, not sure about this, because this now sets the global twice?

Reply:
Not sure to understand, I tested with the case provided in #6326 and it works, normally it modifies the tracing options, with what other global initialization do you think there is a conflict?